services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -21,7 +21,6 @@
 from uuid import UUID
 
 import httpx
-from async_timeout import timeout
```

Comment:
Please remove ``async_timeout`` from the requirements as well to make sure we do not use it anywhere else

Reply:
I will adapt also the dynamic-sidecar (which uses the same library) accordingly in my PR

Reply:
please note that I did not remove async-timeout everywhere. just where it was used wrongly. shall I totally remove it?

Reply:
I see. If you reviewed the cases where it was used wrong, then i guess i tis ok

Reply:
well anyway I removed their usage. but you will not be able to remove the async-timeout library since it comes in through dependencies.